README.md

```diff
@@ -0,0 +1,21 @@
+# Kit Maintenance Docs
+
+This is the maintenance documentation for the Student Robotics team kits.
+
+Each component is documented separately:
+
+ * [Battery](src/battery.md)
+ * [Motor Board](src/motor-board.md)
+ * [Odroid](src/odroid.md)
+ * [Power Board](src/power-board.md)
+ * [Ruggeduino](src/ruggeduino.md)
+ * [Servo Board](src/servo-board.md)
+ * [Usb Charger](src/usb-charger.md)
+ * [Usb Hub](src/usb-hub.md)
+
+## License
```

Comment:
We should add a dedicated `LICENSE` file as well, as with https://github.com/srobo/runbook/blob/master/LICENSE

Reply:
03a0248bbb8c410e4e4b944fa17876fbdc34abd0 does this.